### DISCUSSION TITLE
Will the BE workload be supported on MAC

### DISCUSSION THREAD

**Discussion by dhrumil83** [microsoft/Microsoft-Fabric-workload-development-sample#41] [Q&A] [answered]
Currently the dev setup only seems to work on Windows.

**Comment by ramizsha** [comment, marked as answer]
DevGateway is currently supported only on windows. Not on MAC or Linux.
We're working to add such support by GA (Nov 18).

**Reply by depo34** [reply]
Is a Mac version available now that it has GA?
